bake.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -euo pipefail
+
+OS="${OS-flatcar}"
+FORMAT="${FORMAT:-squashfs}"
+ARCH="${ARCH-}"
+
+# This script is to be called as helper by other scripts but can also be used standalone
+if [ $# -lt 1 ]; then
+  echo "Usage: $0 SYSEXTNAME"
+  echo "The script will make a SYSEXTNAME.raw image of the folder SYSEXTNAME, and create an os-release file in it, run with --help for the list of supported environment variables."
+elif [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "If ARCH is specified as environment variable the sysext image will be required to run on the given architecture."
+  echo "To build for another OS than Flatcar, pass 'OS=myosid' as environment variable (current value is '${OS}'), e.g., 'fedora' as found in 'ID' under '/etc/os-release', or pass 'OS=_any' for any OS."
+  echo "The '/etc/os-release' file of your OS has to include 'SYSEXT_LEVEL=1.0' as done in Flatcar (not needed for 'OS=_any')."
+  echo "If the mksquashfs tool is missing you can pass FORMAT=btrfs, FORMAT=ext4, or FORMAT=ext2 as environment variable (current value is '${FORMAT}') but the files won't be owned by root."
```

Comment:
```suggestion
  echo "If the mksquashfs tool is missing you can pass FORMAT=btrfs, FORMAT=ext4, or FORMAT=ext2 as environment variable (current value is '${FORMAT}') but the script won't change the ownership of the files in SYSEXTNAME directory, so make sure that they are owned by root before creating the sysext image."
```

Reply:
Thanks, the sysext is mounted read-only, but it still makes sense to hint people into setting this up correctly to avoid any problems when a mutable overlay exists.